### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by Speed1**
The box must be unchecked if you WANT users to have to change their password.

### CODE REVIEW

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Follow the steps below if you want to set user passwords to expire after a speci
   
 1. Select **Password expiration policy**.
   
-1. If you don't want users to have to change passwords, uncheck the box next to **Set passwords to never expire**.
+1. If you want users to have to change passwords, uncheck the box next to **Set passwords to never expire**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md:63]:
```suggestion
1. If you want users to be able to change passwords, uncheck the box next to **Set passwords to never expire**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Speed1, kindly commit the suggested change. Many thanks.

**Comment by Speed1** [comment]
Hi @JohanFreelancer9,

sorry for insisting but the option to "be able to change the password" doesn't depend on this flag.

What do you think about this sentence structure?
commit d21c5c0be2046dc3fd9100ede4d5e64a0392e1a9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> Hi @JohanFreelancer9,
> 
> sorry for insisting but the option to "be able to change the password" doesn't depend on this flag.
> 
> What do you think about this sentence structure? commit [d21c5c0](https://github.com/MicrosoftDocs/microsoft-365-docs/commit/d21c5c0be2046dc3fd9100ede4d5e64a0392e1a9)

Hi @Speed1, it is absolutely fine now. Thanks very much.

**Pull request merged**